bigquery/list_datasets_and_projects.js

```diff
@@ -23,11 +23,16 @@
 
 // [START all]
 // [START auth]
-// By default, gcloud will authenticate using the service account file specified
-// by the GOOGLE_APPLICATION_CREDENTIALS environment variable and use the
-// project specified by the GCLOUD_PROJECT environment variable. See
-// https://googlecloudplatform.github.io/gcloud-node/#/docs/guides/authentication
-var gcloud = require('gcloud');
+// By default, the client will authenticate using the service account file
+// specified by the GOOGLE_APPLICATION_CREDENTIALS environment variable and use
+// the project specified by the GCLOUD_PROJECT environment variable. See
+// https://googlecloudplatform.github.io/gcloud-node/#/docs/google-cloud/latest/guides/authentication
+var BigQuery = require('@google-cloud/bigquery');
+
+var Resource = require('@google-cloud/resource');
+
+// Instantiate a resource client
+var resource = Resource();
```

Comment:
Should we put this closer to where it's used? (I can see the merit of leaving it at the top too...)